scripts/update_inventory.py

```diff
@@ -0,0 +1,35 @@
+import docopt
+
+import teuthology
+import teuthology.config
+import teuthology.lock
+import teuthology.orchestra.remote
```

Comment:
The .config import is unnecessary so I'm taking it out. The other two are needed, yes.